maintainers/maintainer-list.nix

```diff
@@ -6563,6 +6563,12 @@
     githubId = 111265;
     name = "Ozan Sener";
   };
+  otavio = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add otavio``.

Reply:
This is already in a specific commit; I reworked the commit log as you suggested.